magic_build_and_package.sh

```diff
@@ -40,17 +42,11 @@ if [ -z "$LANG" ]; then
   LANG='en-US'
 fi
 
-#  for german builds
-# TODO: Use MOZ_UI_LOCALE directly.
-if [[ "$LANG" == 'de' ]]; then
-echo '***** German builds detected *****'
-  IS_DE=true
-  if [[ $IS_MAC_OS ]]; then
-    export L10NBASEDIR=../../l10n  # --with-l10n-base=...
-  else
-    export L10NBASEDIR=../l10n  # --with-l10n-base=...
-  fi
-  export MOZ_UI_LOCALE=de  # --enable-ui-locale=...
+# for localization repack
+if [[ $IS_MAC_OS ]]; then
+  export L10NBASEDIR=../../l10n  # --with-l10n-base=...
+else
+  export L10NBASEDIR=../l10n  # --with-l10n-base=...
 fi
```

Comment:
Looks like this variable used only in language repacking process, so we can add it always. By default LANG=en-US